py2hwsw/lib/hardware/shifters/iob_pack/iob_pack.py

```diff
@@ -5,13 +5,348 @@
+                ],
+            },
+            {
+                "name": "pcnt",
+                "descr": "Push count wire",
+                "signals": [
+                    {
+                        "name": "pcnt",
+                        "width": "2",
+                        "descr": "Push Count Wire",
+                    },
+                ],
+            },
+            {
+                "name": "pcnt_nxt",
+                "descr": "Push count next wire",
+                "signals": [
+                    {
+                        "name": "pcnt_nxt",
+                        "isvar": True,
+                        "width": "2",
+                        "descr": "Push Count Next Wire",
+                    },
+                ],
+            },
+        ],
         "subblocks": [
             {
                 "core_name": "iob_bfifo",
-                "instance_name": "iob_bfifo_inst",
+                "instance_name": "bfifo",
+                "parameters": {
+                    "DATA_W": "DATA_W",
+                },
+                "connect": {
+                    "clk_en_rst_s": "clk_en_rst_s",
+                    "write_io": (
+                        "write_io",
+                        [
+                            "write_i: push",
+                            "wwidth_i: width_i",
+                            "wdata_i: rdata_i",
+                            "wlevel_o: push_level",
+                        ],
+                    ),
+                    "read_io": (
+                        "read_io",
+                        [
+                            "read_i: pop",
+                            "rwidth_i: pop_width",
+                            "rdata_o: wdata_o",
+                            "rlevel_o: pop_level",
+                        ],
+                    ),
+                },
+            },
+            {
+                "core_name": "iob_reg",
+                "instance_name": "wrap_acc_reg",
+                "port_params": {
+                    "clk_en_rst_s": "c_a_r",
+                },
+                "parameters": {
+                    "DATA_W": "$clog2(DATA_W)+1",
+                    "RST_VAL": "{$clog2(DATA_W) + 1{1'b0}}",
+                },
+                "connect": {
+                    "clk_en_rst_s": "clk_en_rst_s",
+                    "data_i": "wrap_acc_nxt",
+                    "data_o": "wrap_acc",
+                },
+            },
+            {
+                "core_name": "iob_reg",
+                "instance_name": "pcnt_reg",
+                "port_params": {
+                    "clk_en_rst_s": "c_a_r",
+                },
+                "parameters": {
+                    "DATA_W": "2",
+                    "RST_VAL": "2'b0",
+                },
+                "connect": {
+                    "clk_en_rst_s": "clk_en_rst_s",
+                    "data_i": "pcnt_nxt",
+                    "data_o": "pcnt",
+                },
             },
```

Comment:
Usually these iob_reg are inferred automatically from comb, since they contain a `_nxt` signal. I don't think we need to define them manually.

Info about iob_comb and inferred registers: https://github.com/IObundle/py2hwsw/blob/14510969c36049a38661146e714939a92128c0ab/py2hwsw/scripts/user_api/api.py#L552-L568